cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -272,6 +266,14 @@ void CacheAllocator<CacheTrait>::initCommon(bool dramCacheAttached) {
       nvmAdmissionPolicy_->initMinTTL(config_.nvmAdmissionMinTTL);
     }
   }
+  if (numTiers_ > 1 && !config_.moveCb) {
+    XLOG(WARN, "No moveCb set, using memcpy for moving items between tiers.");
+    config_.moveCb = [](Item& oldItem, Item& newItem, Item* parentItem){
```

Comment:
Some legacy logic depends on whether `moveCb` is set or not. Are we sure that we are not breaking legacy behavior?